### PR TITLE
upgrade packages

### PR DESCRIPTION
**Pull request by Viktoriia-Bob**

### CODE REVIEW

File: generators/app/templates/src/config/connection/connection.ts [hunk 1]
```diff
@@ -2,17 +2,13 @@ import * as mongoose from 'mongoose';
 import config from '../env/index';
 
 interface IConnectOptions {
-    autoReconnect: boolean;
-    reconnectTries: number; // Never stop trying to reconnect
-    reconnectInterval: number;
-    loggerLevel ? : string;
+    loggerLevel?: any;
```
Comment (v-checha) [inline generators/app/templates/src/config/connection/connection.ts]: remove option loggerLevel

File: generators/app/templates/src/config/connection/connection.ts [hunk 2]
```diff
@@ -2,17 +2,13 @@ import * as mongoose from 'mongoose';
...
+    useUnifiedTopology?: boolean;
```
Comment (v-checha) [inline generators/app/templates/src/config/connection/connection.ts]: remove optional, make !

File: generators/app/templates/src/config/middleware/middleware.ts [hunk 3]
```diff
@@ -60,16 +57,17 @@ export function configure(app: express.Application): void {
...
-            autoReconnect: true
+        store: process.env.NODE_ENV === 'development' ? 
+        new session.MemoryStore() :
+        MongoStore.create({
+            mongoUrl: `${config.database.MONGODB_URI}${config.database.MONGODB_DB_MAIN}`,
         })
     <%_ }_%>
     <%_ if(sessionStore === 'redis') { _%>
-        store: new RedisStore({
+        store: new RedisStore({ client: redis.createClient ({
```
Comment (v-checha) [inline generators/app/templates/src/config/middleware/middleware.ts]: remove space

File: generators/app/templates/src/config/middleware/middleware.ts [hunk 4]
```diff
@@ -60,16 +57,17 @@ export function configure(app: express.Application): void {
...
             port: config.redis.port,
             host: config.redis.host,
-        })
+        })})
```
Comment (v-checha) [inline generators/app/templates/src/config/middleware/middleware.ts]: make br

File: generators/app/templates/src/config/connection/connection.ts (hunk 2 above)
Comment (Viktoriia-Bob) [reply generators/app/templates/src/config/connection/connection.ts]: ok, remove optional

File: generators/app/templates/src/config/middleware/middleware.ts (hunk 3 above)
Comment (Viktoriia-Bob) [reply generators/app/templates/src/config/middleware/middleware.ts]: I'll remove space

File: generators/app/templates/src/config/middleware/middleware.ts (hunk 4 above)
Comment (Viktoriia-Bob) [reply generators/app/templates/src/config/middleware/middleware.ts]: ok, I'll make it

File: generators/app/templates/src/config/connection/connection.ts (hunk 1 above)
Comment (Viktoriia-Bob) [reply generators/app/templates/src/config/connection/connection.ts]: I'll remove loggerLevel

**Pull request merged**